app/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,8 @@
         1. You are supposed to make as many words as you can from the given set of letters in the grid.
         \n2. If the submitted word is a valid one then it is added to the list and the score would be updated.
         \n3. The minimum valid word length is 4 i.e. "cars" is a valid word whereas "car" is not.
+        \n4. You can select letters adjacent to currently selected letter's only.
```

Comment:
Make this the 2nd point and `letter's` should be `letters`.  Also adjacent is not a correct term as it doesn't represent squares sharing a vertex.